src/CoreBundle/private/library/classes/TCMSURLHistory.class.php

```diff
@@ -21,7 +21,17 @@ class TCMSURLHistory
      */
     public $aHistory = array();
 
-    public $index = 0;
+    /**
+     * @deprecated since 6.3.0 - use getHistoryIndex() instead
+     */
+    // public $index = 0;
+
+    public function __get($parameterName)
```

Comment:
We should also add a `__set` method that reacts on setting `index` as it was a public property before this change and clients could still set it. Maybe triggering a warning is the right thing to do there.

Reply:
done, also added __isset